### PR TITLE
refactor(console): settings

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Requirements:
https://www.figma.com/file/hqAWH3Di8gkiV5TXAt6juO/%F0%9F%8C%B9-%5BAC%5D-Layout-UI-Optimization?node-id=174%3A23805&t=bKjlee8sZDBEwfTV-4

image

- refactor: settings page layout
- adjust the `<ChangePassword />` component styles to fit the new layout.

### Before
image

### After
image

### CODE REVIEW

File: packages/console/src/pages/Settings/components/ChangePassword.module.scss [hunk 1]
```diff
@@ -6,7 +6,8 @@
   padding: _.unit(4);
   display: flex;
   align-items: center;
-  width: 554px;
+  justify-content: space-between;
+  @include _.form-text-field;
```
Comment (simeng-li) [inline packages/console/src/pages/Settings/components/ChangePassword.module.scss:10]: nit: just curious, is `changePassword` a form text field?

File: packages/console/src/pages/Settings/index.tsx [hunk 2]
```diff
@@ -49,17 +48,21 @@ const Settings = () => {
...
   return (
-    <Card className={classNames(detailsStyles.container, styles.container)}>
-      <CardTitle title="settings.title" subtitle="settings.description" />
-      <TabNav>
-        <TabNavItem href="/settings">{t('settings.tabs.general')}</TabNavItem>
-      </TabNav>
+    <div className={classNames(resourcesStyles.container, styles.container)}>
+      <div className={resourcesStyles.headline}>
+        <CardTitle title="settings.title" subtitle="settings.description" />
+      </div>
```
Comment (simeng-li) [inline packages/console/src/pages/Settings/index.tsx:54]: Could we apply the headline styles directly to the CardTitle instead of wrap in a extra wrapper?

Comment (xiaoyijun) [reply packages/console/src/pages/Settings/index.tsx:54]: updated.

File: packages/console/src/pages/Settings/components/ChangePassword.module.scss (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/Settings/components/ChangePassword.module.scss]: updated.

File: packages/console/src/pages/Settings/index.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/Settings/index.tsx:54]: No, the CardTitle has its own structure and layout styles, if we apply the headline styles directly, the `<CardTitle />`'s looking will be wrong.

Comment (xiaoyijun) [reply packages/console/src/pages/Settings/index.tsx]: The div wrapper is removed for it's unnecessary.

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**